app/src/main/java/ch/epfl/sweng/studyup/utils/adapters/QuestListViewAdapterStudent.java

```diff
@@ -0,0 +1,54 @@
+package ch.epfl.sweng.studyup.utils.adapters;
```

Comment:
This is an old file not used anymore (it's a nested class now), it should be deleted